tensorflow/buildspec.yml

```diff
@@ -53,30 +55,30 @@ images:
       *DEVICE_TYPE ]
     context:
       <<: *TRAINING_CONTEXT
-  BuildTensorflowGpuPy37Cu112TrainingDockerImage:
+  BuildTensorflowGpuPy37Cu110TrainingDockerImage:
     <<: *TRAINING_REPOSITORY
     build: &TENSORFLOW_GPU_TRAINING_PY3 false
-    image_size_baseline: &IMAGE_SIZE_BASELINE 7738
+    image_size_baseline: &IMAGE_SIZE_BASELINE 10352
```

Comment:
why is the baseline 10G? We never crossed 10G in earlier releases

Reply:
Error snippet from codebuild logs:
<img width="1140" alt="Screen Shot 2021-07-23 at 2 21 06 PM" src="https://user-images.githubusercontent.com/22492939/126842772-a2440fd4-f705-4add-9ed4-7bf2bc0c61f3.png">

I changed image size baseline based on these numbers, is that not how  baseline is set?